src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -106,6 +106,16 @@
         total-units-in-all-chains (total-token-units-in-all-chains token)]
     (money/crypto->fiat total-units-in-all-chains price)))
 
+(defn total-network-fiat-value
+  "Returns the total network fiat value."
```

Comment:
this is just for a single asset? or for all assets on a network?

Reply:
ah yeah but it's for a single token. - maybe just update the description you have with that info?